src/main/java/uk/gov/ons/census/action/client/CaseClient.java

```diff
@@ -16,13 +16,13 @@
   private String port;
 
   public UacQidDTO getUacQid(UUID caseId, String questionnaireType) {
+    //    Should be URIBuilder
```

Comment:
so why isn't it?

Reply:
sugar,  meant to remove this comment as was out of scope for this ticket.  Will fix it